IreneBranch/Project1

```diff
@@ -0,0 +1 @@
+Subproject commit b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996
```

Comment:
ok, well done! but now the branch has merge conflicts. Not sure why though.

 Did you update the master branch after your branched off to this branch?
Try to use the command line to resolve the issue, as github suggests.

On Mon, May 2, 2016 at 9:33 PM, IreneG5 <notifications@github.com> wrote:

> In IreneBranch/Project1
> <https://github.com/IreneG5/Project1/pull/2#discussion_r61781311>:
>
> > @@ -0,0 +1 @@
> > +Subproject commit b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996
>
> done! (I think)
> … <#m_-8547804738445428477_>
> On 2 May 2016 at 19:22, Ariadni-Karolina Alexiou ***@***.***> wrote: In
> IreneBranch/Project1 <#2 (comment)
> <https://github.com/IreneG5/Project1/pull/2#discussion_r61779610>>: > @@
> -0,0 +1 @@ > +Subproject commit b9f80c0
> <https://github.com/IreneG5/Project1/commit/b9f80c09e9ca3f8f2bb55e9b79e78a0e2bcb6996>
> git rm this folder from your branch — You are receiving this because you
> authored the thread. Reply to this email directly or view it on GitHub <
> https://github.com/IreneG5/Project1/pull/2/files/ea58a2dc7f0a59d8ae8da51f7d79e2a432cab3a1#r61779610>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/IreneG5/Project1/pull/2/files/ea58a2dc7f0a59d8ae8da51f7d79e2a432cab3a1#r61781311>
>